### PR TITLE
Don't hardcode branch name for docbuild

### PR DESCRIPTION
**Pull request by simaishi**
Pass in branch name (but default to 'master' if not passed), so this file doesn't need to be edited for each branch.

### CODE REVIEW

**Comment by simaishi** [comment]
@Fryguy please review.

File: scripts/docbuild.rb [hunk 1]
```diff
@@ -7,7 +7,7 @@
 BUILD_BASE     = Pathname.new("/build")
 FILESHARE_DIR  = BUILD_BASE.join("fileshare")
 
-DOCS_BRANCH    = "master"
+DOCS_BRANCH    = ARGV[0].nil? ? "master" : ARGV[0]
```
Comment (abellotti) [inline scripts/docbuild.rb]: would prefer:  ARGV[0] || "master"

Thanks.

Comment (simaishi) [inline scripts/docbuild.rb]: Cleaner that way... done. Thanks!

**Comment by Fryguy** [comment]
@simaishi You shouldn't need the branch at all.  asciibinder, by default, builds *all* of the branches.  You can build branch by branch if you want, so I'm not sure what you prefer.

**Comment by Fryguy** [comment]
@simaishi I reviewed the current stuff, and I think it's better to remove the branches entirely.  asciibinder should not do a git checkout before a build because it needs things from the master branch (like some of the config files).

**Comment by simaishi** [comment]
Wasn't aware it builds all branches by default. If that's the case, I think we can move up the `latest-docs` directory one level up (currently it's under 'master' directory), and put each branch doc to respective branch directory (which is related to https://github.com/ManageIQ/manageiq-appliance-build/pull/133).

**Comment by simaishi** [comment]
@Fryguy @abellotti ascii_builder builds all branches in "local" git repo. So we need to make sure all needed branches exist locally, not just 'master'.  (Which is why I wasn't seeing 'euwe' being built after modifying distro_map yml)

So we can either hard-code needed branches somewhere, or build per branch from each nightly as per my original thought (by passing branch name).  Or maybe some other options?

**Comment by Fryguy** [comment]
you should be able to `git fetch --all`, then run it.  That will get the branch locally, and the distro map yml is what drives the creation of the directories.

**Pull request closed without merging**